testenvironments.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1",
+    "environments": [
+      {
+        "name": "WSL-Ubuntu",
+        "type": "wsl",
+        "wslDistribution": "Ubuntu"
+      },
+      {
+        "name": "docker dotnet 7.0",
```

Comment:
I imagine this would ideally get updated at times, and I'm somewhat concerned it's going to be forgotten until something breaks—probably a long time from now. It would be neat to have a build step that pulls `7.0` from our global.json, but that sounds like overengineering a very minor thing.

Reply:
Correct, I had same line of thoughts.
But let's wait to see whether this is going to be used, then we can improve.